Sources/Interaction/UI/VolumeController/index.js

```diff
@@ -303,6 +311,10 @@ export function extend(publicAPI, model, initialValues = {}) {
   macro.setGet(publicAPI, model, ['actor', 'renderWindow', 'rescaleColorMap']);
   macro.get(publicAPI, model, ['widget']);
 
+  // Add user custom presets if they exist.
+  if (Object.prototype.hasOwnProperty.call(initialValues, 'userLookupTable')) {
```

Comment:
I don't think as UI widget should modify the vtkColorMaps. This should be managed at the application level.